### PR TITLE
🔀🔖 4.6.3

### PR DESCRIPTION
**Pull request by jxn-30**
**Releasenotes for this release**
* 🌐 many translation updates
* 💚 many changes and updates in the background
* ✨ [generalExtensions] map bookmarks: custom titles (#1791)
* ✨ [extendedBuilding] dispatchCenterBuildingFilter
* ✨ [extendedBuilding / schoolsBuildingFilter] add filter by dispatch center
* ✨ [hotkeys] many new hotkeys for buildings and dispatch centers (thanks to @Cr4zyc4k3)
* ✨🇫🇷 CP fr_FR 🇫🇷✨

---
**Releasenotes for translation**
```md
* 🌐 many translation updates
* 💚 many changes and updates in the background
* ✨ [$m('generalExtensions')] map bookmarks: custom titles (#1791)
* ✨ [$m('extendedBuilding')] dispatchCenterBuildingFilter
* ✨ [$s('extendedBuilding.schoolsBuildingFilter')] add filter by dispatch center
* ✨ [$m('hotkeys')] many new hotkeys for buildings and dispatch centers (thanks to @Cr4zyc4k3)
* ✨🇫🇷 CP fr_FR 🇫🇷✨
```

### CODE REVIEW

**Review by Suchty112 — APPROVED** [review]

File: src/userscript.ts [hunk 1]
```diff
@@ -10,7 +10,9 @@ const loadLSSM = () => {
 
     script.src = `${host}core.js?_=${new Date().getTime()}&uid=${
         I18n.locale
-    }-${user_id}`;
+    }-${user_id}&branch=${
+        localStorage.getItem(`${prefix}_branch`) ?? 'stable'
```
Comment (Cr4zyc4k3) [inline src/userscript.ts:14]: What's the prefix by default?

File: src/modules/redesign/components/vehicle_group.vue [hunk 2]
```diff
@@ -144,6 +144,8 @@ export default Vue.extend<
     },
     mounted() {
         this.lightbox.finishLoading('vehicle_group-mounted');
+        // eslint-disable-next-line no-console
+        console.log(this.$refs.caption, this.$refs.color, this.$refs.column);
```
[the remark this answers is not in the document]
Comment (jxn-30) [reply src/modules/redesign/components/vehicle_group.vue:148]: yes, that part of Redesign is still WIP

File: src/userscript.ts (hunk 1 above)
Comment (jxn-30) [reply src/userscript.ts:14]: it is `lssmv4` and will be set when building the userscript.

File: src/modules/hotkeys/assets/commands/building.ts [hunk 3]
```diff
@@ -0,0 +1,274 @@
+import type { Empty, Scope } from 'typings/modules/Hotkeys';
```
[the remark this answers is not in the document]
Comment (jxn-30) [reply src/modules/hotkeys/assets/commands/building.ts:1]: it looked good to me ;)

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**